web/styles/compose.css

```diff
@@ -173,21 +173,15 @@
     height: 100%;
 }
 
-#below-compose-content {
-    display: flex;
-    flex-direction: column;
-    width: 100%;
+.compose_bottom_top_container {
     margin-top: 6px;
-    margin-bottom: -2px;
-
-    .compose_bottom_top_container {
-        display: flex;
-    }
+    display: flex;
+}
 
-    .compose_bottom_bottom_container {
-        display: flex;
-        justify-content: space-between;
-    }
+.compose_bottom_bottom_container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: -2px;
```

Comment:
I'd love to figure out why this negative bottom margin is needed, and eliminate it if possible.

Reply:
(Otherwise, this might be something that just gets cleaned up when we start rewriting the compose box as grid--so don't work too hard at it.)